src/model/model.rs

```diff
@@ -37,11 +37,14 @@ use std::str::FromStr;
 use derive_more::{Display, From, Into};
 use failure::Fail;
 use log::*;
-use regex::Regex;
+use regex::{escape, Regex};
 use serde::*;
 
 pub const DEFAULT_PROTOCOL_VERSION: ProtocolVersion = ProtocolVersion(5);
 
+const WILDCARD_AFTER_ESCAPE: &'static str = r"\*";
+const WILDCARD_REPLACEMENT: &'static str = r"[a-zA-Z0-9-]{1,63}";
```

Comment:
Please change these names to more readable names. These names are given from their constructions nor than means.
For example, `IPV4_ADDRESS` is preferable rather than `FOUR_EIGHT_BIT_HEX_DIGITS_SEPARATED_BY_DOT`.

And you may define it locally within `AddressPattern::match` like:

```
{
    let WILDCARD_AFTER_ESCAPE = r"\*";
    let pattern = format!(
        r"\A{}\z",
        &escape(s).replace(WILDCARD_AFTER_ESCAPE, WILDCARD_REPLACEMENT)
    );
}
```

Reply:
done with 3a45b24, b2baa50